json_serializable/lib/src/nested_field_utils.dart

```diff
@@ -0,0 +1,23 @@
+class NestedUtils {
```

Comment:
put these at the root of the file. No need to wrap them in a class as static functions